### ISSUE TITLE
There is a problem with the annotate module

### ISSUE THREAD

**Issue by 002-two** [linzhi2013/MitoZ#112]
Hi,
I assemble and annotated an complete mitochondrial using MitoZ 2.3， because of genetic rearrangement，I used this script （Mitogenome_reorder.py）to adjust the order.  The ordered file is generated--test_all.fasta.reorder, I converted it to a fasta file. 

An error occured when I want to annotate again with the annotate module.
![7ae986b642e0c9f1a7c31f08f7f6e64](https://user-images.githubusercontent.com/77915435/105625016-2d502d80-5e61-11eb-85df-0ec83a0ce889.png)

The path here is the same as in the all2 module I used. So ,I don`t know how to solve it.
This is annotate script.
# make sure your fq1 and fq2 files are in $PWD directory!
PWD=/data/test
time docker run -v $PWD:$PWD -w $PWD  --rm guanliangmeng/mitoz:2.3 \
python3 /app/release_MitoZ_v2.3/MitoZ.py annotate \
--genetic_code 2 \
--clade Chordata \
--outprefix test_annotate \
--thread_number 8 \
--fastafile test_all.reorder.fasta \
1>test_annotate.log 2>test_annotate.err

**Comment by linzhi2013** [comment]
Add the slash at the end of each line, if they belong to the same command.

```
PWD=/data/test
time docker run -v $PWD:$PWD -w $PWD --rm guanliangmeng/mitoz:2.3 \
python3 /app/release_MitoZ_v2.3/MitoZ.py annotate \
--genetic_code 2 \
--clade Chordata \
--outprefix test_annotate  \
--thread_number 8 \
--fastafile test_all.reorder.fasta \
1>test_annotate.log 2>test_annotate.err
```

**Comment by 002-two** [comment]
Hi，
I originally had slashes in my script. It can't have been lost when I copied it here.
![b4fcd64546e9a5f3bbd05d846f6469c](https://user-images.githubusercontent.com/77915435/105787727-08bb9900-5fba-11eb-8e09-b99dfa52248d.png)
I tried running it again, and the same error returned.
![089dedf3fe74aa340ab64ba8b0bd855](https://user-images.githubusercontent.com/77915435/105787846-3f91af00-5fba-11eb-9108-859b13400040.png)
